faq.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Responsive grid project</title>
+        <link rel="stylesheet" href="./css/header.css"/>
+        <link rel="stylesheet" href="./css/faq.css"/>
+        <link rel="stylesheet" href="./css/footer.css"/>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Fjalla+One&family=Outfit:wght@200&display=swap" rel="stylesheet">
+        
+    </head>
+
+    <body>
+        <header class="header">
+            <div class="header-top-half">
+                <!-- SEARCH BAR -->
+                <div class="search-container">
```

Comment:
This div could actually be a form. There are some other attributes you can use to ensure an input of type `search` is used in an accessible way, check out the docs https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search